.github/workflows/ci.yaml

```diff
@@ -98,17 +107,17 @@ jobs:
         path: dist/*.whl
         if-no-files-found: error
 
-    - name: Install python dependencies
-      run: ./dev.py install-python-deps
-
-    - name: Install wheel packages
-      run: ./dev.py install-wheel-packages
+    - name: Set up Docker Buildx
+      if: matrix.node_version == '19.x'
+      uses: docker/setup-buildx-action@7703e82fbced3d0c9eec08dff4429c023a5fd9a9
```

Comment:
Can you add human-readable version in the comments?
https://github.blog/changelog/2022-10-31-dependabot-now-updates-comments-in-github-actions-workflows-referencing-action-versions/